lib/opencensus/trace/formatters/binary.rb

```diff
@@ -28,6 +28,22 @@ class Binary
         #
         BINARY_FORMAT = "CCH32CH16CC".freeze
 
+        ##
```

Comment:
This formatter will malfunction if set as the formatter for faraday. It should probably not appear to be of the same duck type, so as to fail faster.